### DISCUSSION TITLE
Building instance segmentation model

### DISCUSSION THREAD

**Discussion by NoaMills-USDA** [azavea/raster-vision#1498] [Q&A] [answered]
I have a functional semantic segmentation model with 4 bands of satellite image input. I'm interested in building an instance segmentation model based on the same data. Is this possible with version 0.13.1? I saw information about TaskConfig for version 9.0, but I can't seem to find something equivalent in version 0.13.1. An example of a functional instance segmentation model would be really helpful!

**Comment by AdeelH** [comment, marked as answer]
Raster Vision does not currently support instance segmentation, but we have been interested in adding support for it for a while (#854). 

If you need this for official USDA work, there might be an opportunity here to have the USDA fund this work (i.e. adding support of instance segmentation to RV). If you are interested in exploring this possibility we would be eager to chat. If you share your email address (via a Gitter DM) or email me at ---, I can connect you with our business development team to further discuss this.

**Reply by NoaMills-USDA** [reply]
Thank you for your response. I think we'll be able to make a semantic segmentation model work for our use case.
